docs/react-v9/contributing/common-dev-snags.md

```diff
@@ -6,11 +6,11 @@ First, file an issue [here](https://github.com/microsoft/fluentui/issues/new/cho
 
 Make sure all your desired changes are stashed or otherwise saved somewhere you can pull them back in.
 
-First try running:
+Then try running:
 
 `yarn`
 
 If the issues still persist, clean your repo:
 
 `yarn cache clean`
-`yarn`
+`yarn` // Vanilla Windows users may have better luck with yarn build.
```

Comment:
can we make obvious what "vanilla windows" means when this is part of official docs ?